app/core/Tracker2/DriftTracker.js

```diff
@@ -65,24 +78,31 @@ export default class DriftTracker extends BaseTracker {
   async identify (traits = {}) {
     await this.initializationComplete
 
-    if (this.store.getters['me/isAnonymous']) {
-      return
-    }
-
     const { me } = this.store.state
 
     const {
-      id,
+      _id,
       ...meAttrs
     } = me
 
-    await window.drift.identify(
-      me._id.toString(),
-      {
-        ...meAttrs,
-        ...traits
-      }
-    )
+    const filteredMeAttributes = Object.keys(meAttrs)
+      .reduce((obj, key) => {
+        if (DEFAULT_DRIFT_IDENTIFY_USER_PROPERTIES.includes(key) && meAttrs[key] !== null) {
```

Comment:
This line made me check how many users use internet explorer and turns out in the last week only 23 users or 0.02% users. So that is good to know!